.github/workflows/build.yml

```diff
@@ -355,8 +355,8 @@ jobs:
 
       - name: Update build version and number
         run: |
-          yq -i '.settings.MARKETING_VERSION = "${{ steps.version_info.outputs.version_name }}"' 'project.yml'
-          yq -i '.settings.CURRENT_PROJECT_VERSION = "${{ steps.version_info.outputs.version_number }}"' 'project.yml'
+          yq -i '.settings.MARKETING_VERSION = "${{ steps.version_info.outputs.version_name }}"' 'project_passwordmanager.yml'
+          yq -i '.settings.CURRENT_PROJECT_VERSION = "${{ steps.version_info.outputs.version_number }}"' 'project_passwordmanager.yml'
```

Comment:
🤔 As far as I can see we're using hyphen `-` instead of underscore `_` to separate words in the filenames for `yml` like `crowdin-pull.yml` or `github-release.yml`. Is there a reason to change to underscore for the new ones in this PR?

Reply:
Both of those seem very reasonable to me; when I wrote the PM section of the [Action Plan](https://bitwarden.atlassian.net/wiki/spaces/MNAE/pages/1336868878/Mobile+Monorepos+Action+Plan) I was patterning off of what @vvolkgang had done for the BWA section, but I think hyphens and abbreviations is a good way to go. Álison, what do you think?

If we do this with hyphens and abbreviations, I can get the Action Plan updated to match.